tools/pythonpkg/tests/fast/udf/test_null_filtering.py

```diff
@@ -106,8 +106,8 @@ def get_types():
         ),
         Candidate(
             BLOB,
-            b'\xF6\x96\xB0\x85',
```

Comment:
Could you expand on why these files are changed?

Reply:
These diffs are due to running `make format-fix`; I'm happy to revert the changes here if you'd prefer.